File: Modern/src/main/scala/prv/zielony/transaction/analyst/modern/events/accounts/AccountEvent.scala
package prv.zielony.transaction.analyst.modern.events.accounts

import java.util.Date

import prv.zielony.transaction.analyst.modern.events.{Identifier, Event}

/**
 * Created by zielony on 14.02.16.
 */
abstract class AccountEvent(timestamp:Date, accountId:Long) extends Event(timestamp) with Identifier[Long] {
  override val id:Long = accountId;
}

File: Classical/src/main/java/prv/zielony/transaction/analyst/classical/dto/AccountEvent.java
package prv.zielony.transaction.analyst.classical.dto;

/**
 * Created by zielony on 27.02.16.
 */
public class AccountEvent {

    private Long id;

    private CrudOperation operation;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public CrudOperation getOperation() {
        return operation;
    }

    public void setOperation(CrudOperation operation) {
        this.operation = operation;
    }
}
